lib/gui/models/flash-state.js

```diff
@@ -132,8 +133,8 @@ FlashState.service('FlashStateModel', function() {
         speed: _.attempt(() => {
           if (_.isNumber(state.speed) && !_.isNaN(state.speed)) {
 
-            // Transform bytes to megabytes preserving only two decimal places
-            return Math.floor(state.speed / 1e+6 * 100) / 100;
+            // Preserve only two decimal places
+            return Math.floor(units.bytesToMegabytes(state.speed) * 100) / 100;
```

Comment:
Just out of curiosity, why is this precision-trimming done here, and not later when the value is actually used / displayed?